javascript/deploynaut.js

```diff
@@ -275,4 +275,23 @@
 		});
 
 	});
+
+	$('button.abort').click(function(event) {
```

Comment:
`on('click', function(event) {` would be better here, as that's recommended over calling click directly.